### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by tpetrone** [fenichelar/ember-simple-auth-token#252]
It would be nice to explicitly inform that when in `test` environment is mandatory to use the following options:
```
    ENV['ember-simple-auth-token'] = {
      ...
      refreshAccessTokens: false,
      tokenExpirationInvalidateSession: false
    };
```
Otherwise `later()` (Ember's run loop function) will cause acceptance tests to halt and eventually time out depending on token's lifetime (server side) and test's timeout configurations.

**Comment by fenichelar** [comment]
@tpetrone Good point! I will add this :)

**Comment by fenichelar** [comment]
Closed by #253.
